www/podj00/sp2/assets/js/main.js

```diff
@@ -0,0 +1,383 @@
+$(document).ready(() => {
+    ///////////
+    //BUTTONS
+    ///////////
+    const removeAll = $('#removeAll');
+    const removeOne = $('#removeOne');
+    const loadButton = $('#load');
+    const selectButton = $('#select');
+    $('#from').datepicker({
+        dateFormat: 'yy-mm-dd',
+        firstDay: '1',
+        changeMonth: true,
+        changeYear: true,
+        minDate: new Date(2020, 1 - 1, 22),
+        maxDate: new Date(),
+        inline: true,
+        onSelect: function (dateStr) {
+            $('#to').datepicker('option', 'minDate', dateStr);
+        }
+    });
+    $('#to').datepicker({
+        dateFormat: 'yy-mm-dd',
+        changeMonth: true,
+        changeYear: true,
+        minDate: new Date(2020, 1 - 1, 22),
+        maxDate: new Date(),
+        inline: true,
+        firstDay: '1'
+    });
+    const proxyUrl = 'https://cors-anywhere.herokuapp.com/';
+    let mockedResponse = {};
+    let mockedOneCountry = {};
+
+    $('input[type="checkbox"]').on('change', (e) => {
+        //only 1 input (clicked one) can be checked
+        $('input[type="checkbox"]').not(e.currentTarget).prop('checked', false);
+        //cannot uncheck - at least one has to be checked
+        $(e.currentTarget).prop('checked', true);
+    });
+
+    ////////
+    //ACTIONS
+    ////////
+    loadButton.click(() => {
+        const section = $('#allCountries');
+        let loader = $('<div class="loader">Loading...</div>');
+        section.append(loader);
+        if (section.children().length > 1) {
+            section.children('canvas').remove()
+        }
+        return getAllCases();
+        /*MOCK CALL
+           return fetch("/podj00/sp2/assets/js/mockedResponse.json").then((response) => response.json()).then(json => {
+            mockedResponse = json;
+            return getAllCasesMocked()
+        })
+         */
+    })
+
+    selectButton.click((e) => {

```

Comment:
udelejte prevenci vicenasobneho kliknuti na toto tlacitko (muze byt napr. animace v tlacitku, a behem nacitani zakazat kliknuti, a po skonceni volani ajaxu to zmenit zpatky do puvodniho stavu)